lib/vendors/axis-communications/sv_vendor_axis_communications.c

```diff
@@ -29,7 +29,7 @@
 #include <stdbool.h>
 #include <stdlib.h>  // malloc, memcpy, calloc, free
 
-#include "sv_internal.h"  // signed_video_t
+#include "sv_internal.h"  // signed_video_t, onvif_media_signing_set_signing_key_pair()
```

Comment:
AFAIK, `onvif_media_signing_set_signing_key_pair()` is not declared in `sv_internal.h`. Include the right header file.

Also, you use the function `port_settings_to_onvif()`. From where is that included?